spec/std/int_spec.cr

```diff
@@ -537,6 +537,18 @@ describe "Int" do
     end
   end
 
+  {% if compare_versions(Crystal::VERSION, "0.26.1") > 0 %}
+    it "compares equality and inequality of signed vs. unsigned integers" do
+      x = -1
+      y = x.unsafe_as(UInt32)
+
+      (x == y).should be_false
+      (y == x).should be_false
+      (x != y).should be_true
+      (y != x).should be_true
```

Comment:
Wouldn't it be better to use `should eq` and `should_not eq` expectations to have a report of variables if it fails?

Reply:
I don't know, here we are testing those operators and I wouldn't like to hide them behind a spec matcher.

Reply:
this is better as-is